src/ai/backend/manager/models/group.py

```diff
@@ -111,6 +114,11 @@ class AssocGroupUserRow(Base):
     group = relationship("GroupRow", back_populates="users")
 
 
+class ProjectType(str, enum.Enum):
```

Comment:
Use `enum.StrEnum` as of Python 3.11 and for the new codes that won't be backported to 22.09 or older.